agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -1,6 +1,6 @@
 - Feature Name: MPID Messaging System
 - Type: New Feature
-- Related Components: [safe_vault](https://github.com/maidsafe/safe_vault), [safe_client](https://github.com/maidsafe/safe_client), [routing](https://github.com/maidsafe/routing)
+- Related Components: [safe_vault](https://github.com/maidsafe/safe_vault), [safe_client](https://github.com/maidsafe/safe_client), [mpid_messaging](https://github.com/maidsafe/mpid_messaging)
 - Start Date: 22-09-2015
```

Comment:
this link https://github.com/maidsafe/mpid_messaging opens nothing.
and, are we going to have a separate crate for messaging? instead of having a persona mpid_manager inside vault?